packages/odyssey-react/src/components/Select/Select.module.scss

```diff
@@ -231,16 +220,27 @@ $choices-icon-cross-inverse: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjEi
     padding-inline: 0;
     opacity: 1;
     background-color: cv('blue', '400');
-    background-image: get-icon('close', $white);
-    background-position: center;
-    background-size: ($base-line-height * 1em) - 1em; // funky math to evade half-pixel rounding errors
     line-height: 1;
 
     &:hover,
     &:focus {
       background-color: $color-primary-base;
     }
   }
+
+  .buttonIndicator {
+    display: block;
+    position: absolute;
+    inset-block-start: 50%;
+    inset-inline-start: 50%;
+    width: 8px;
```

Comment:
Does `$choices-button-dimension` match the use case here or are the values coincidentally the same rn?

Reply:
Yes it does match and those vars are still present and unused in the file... so I could remove them or use them here instead of the fixed value.

Do we want folks to be able to change the close button size?

Reply:
The safest route might be to associate this with the type size, so it scales if people theme that. Maybe `0.5em`?

If we don't want to futz with that, I think it'd be good to use the variables here, so the magic numbers are top-of-file.